src/iiiftilesource.js

```diff
@@ -134,7 +134,7 @@ $.IIIFTileSource = function( options ){
 
     if (!options.maxLevel && !this.emulateLegacyImagePyramid) {
         if (!this.scale_factors) {
-            options.maxLevel = Number(Math.ceil(Math.log(Math.max(this.width, this.height), 2)));
+            options.maxLevel = Number(Math.round(Math.log(Math.max(this.width, this.height), 2)));
```

Comment:
I suspect this one should stay `ceil`. This is determining how many levels we have, so it makes sense to round up. The other ones all have to do with pixel width/height, so they are a different thing.

Reply:
I tested the updated code and it works as expected - no drop in the last tile column.

I ran the test suite and there were no errors concerning the iiiftilesource.js

I not completely sure about the line you have mentioned either but I have not seen any misbehaviour.

Reply:
The other option uses round as well: https://github.com/openseadragon/openseadragon/blob/ce45e34265d27de835ae40e6b486deb14f70c6ba/src/iiiftilesource.js#L140

So IMHO it kinda makes sense to unify the approach...

Reply:
Oh! Interesting... I didn't realize the other path was using round already. @azaroth42 What do you think?

Reply:
Apologies, work-work is snowing me at the moment. Perhaps @tomcrane ?

Reply:
@azaroth42 Thanks for the lead! If I don't hear from @tomcrane in a while, I'll ping you again.